examples/VB_ASR/infer_constitutive_law_ASR_RCA.py

```diff
@@ -119,11 +119,19 @@ def me_scalar(X):
 # VARIATIONAL BAYESIAN (VB)
 ###############################
 
+### Prior parameters
 prior_mvn = MVN(mean=X0, precision=np.diag([(abs(x)) ** (-2) for x in X0]))
-n0 = Gamma(shape=1.5, scale=175000.0)
-# n0 = Gamma(shape=0.84, scale=15000.)
-# n0 = Gamma(shape=0.5, scale=500000.)
-# n0 = Gamma(shape=0.84, scale=370000.)
+
+### Prior noise
+## (1): from quantiles of expected standard deviation
+n0_std_min = 0.002
+n0_std_max = 0.02
+n0 = Gamma.FromSDQuantiles(n0_std_min, n0_std_max, (0.02, 0.98))

```

Comment:
How did you choose the n0_std_min, max, 0.02, 0.98?

Reply:
well, about the 'n0_std_min' and 'n0_std_max', honestly I set them from the other report :). But probably a more realistic way is to take a non-informative Gamma prior. Right?
The Quantiles (0.02, 0.98) are not the only choice. The default in Bayem is (0.05, 095), but I was somehow more confortable with a wider interval. (0.01, 0.99) could also be a choice.

Reply:
other report is the other group that has done the Bayes inference?

Reply:
Yes